### ISSUE TITLE
Cocoapods only installs till 0.3.6

### ISSUE THREAD

**Issue by balintvizeli** [johnpatrickmorgan/FlowStacks#88]
Hey,

I'm getting error when I try to install the library with cocoapods. When I specify the version it says that there's no version like that, if I don't specify it installs 0.3.6.

Error when specifying version:
```
[!] CocoaPods could not find compatible versions for pod "FlowStacks":
  In Podfile:
    FlowStacks (~> 0.7)

None of your spec sources contain a spec satisfying the dependency: `FlowStacks (~> 0.7)`.

You have either:
 * mistyped the name or version.
 * not added the source repo that hosts the Podspec to your Podfile.

```

When not specifying:
`Installing FlowStacks 0.3.6`

**Comment by balintvizeli** [comment]
<img width="1840" alt="Screenshot 2024-10-24 at 10 31 44" src="https://github.com/user-attachments/assets/9cc872b8-cd1a-4195-b015-b50c0f3c2e59">
In cocoapods on 0.3.6 is specified. I cannot update to latest ones. @johnpatrickmorgan

**Comment by johnpatrickmorgan** [comment]
Thanks for pointing this out @balintvizeli, I've now published the pod spec for version 0.8.3. Apologies, I was having issues with my cocoapods installation.
